package.json

```diff
@@ -86,4 +86,4 @@
     ]
   },
   "snyk": true
-}
+}
```

Comment:
Any reason to remove this EOL?

Reply:
No, its been removed by the bump scripts/ or the npm lint looks like? 
Lets me test and maybe fix this as well.

Reply:
If `npm run bump` or `npm run lint` does remove the EOL then let's keep them that way then :)

If they don't then let's bring the EOL back.